eth_crowdfund_be/eth_crowdfund_api/src/tests/test_campaign.py

```diff
@@ -0,0 +1,155 @@
+import pdb
+import unittest 
+import os 
+import json 
+# import flask_sqlalchemy
+from ..models.Campaign import Campaign
+from ..app import create_app, db
+
+def func(x):
+  return x + 1
+# basic test
+def test_answer():
+  assert func(3) == 4
+
+class CampaignTest(unittest.TestCase):
+  # set up the tests with 2 campaigns
+  # This runs before EVERY test. not once before all
+  def setUp(self):
```

Comment:
There are some comments through out that should help with readability. This setup will run before each test